projects/admin/src/app/circulation/patron-transaction.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2019 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* tslint:disable */
```

Comment:
It could be better to disable a specific rule, instead of all rules.

Reply:
According to other 'class' classes.
Removing this rule cause problem to `Object.assign` method (mapping problem between SnakeCase from RERO_ILS and CamelCase from RERO_ILS_UI)